code/game/objects/items/weapons/null_rod.dm

```diff
@@ -173,6 +173,29 @@
 	item_state = "katana"
 	fluff_pickup = "bisect"
 
+/obj/item/weapon/nullrod/crucifix
+	name = "crucifix"
+	desc = "A shiny gold crucifix."
+	icon_state = "crucifix"
+	item_state = "crucifix"
+	fluff_pickup = "turn"
+
+/obj/item/weapon/nullrod/crucifix/pickup(mob/living/carbon/human/user)
+	if(user.mind.assigned_role == "Chaplain")
+		desc = "What you are about to do has not been approved by the Space Vatican."
+	else
+		desc = initial(desc)
```

Comment:
Overwrite examine() here instead to use a different description based on whether or not the item is held by the chaplain. I can give you the code but it really isn't a difficult change so try it yourself first!